src/main/java/com/rojel/wesv/WorldEditSelectionVisualizer.java

```diff
@@ -155,13 +172,10 @@ public void setParticlesForPlayer(final Player player, final Collection<Location
 			final int fade = config.getParticleFadeDelay();
 
 			if (fade > 0) {
-				final int id = this.getServer().getScheduler().runTaskLater(this, new Runnable() {
+				final int id = this.getServer().getScheduler().runTaskLater(this, () -> {
```

Comment:
will this work on Java 7 with older servers?

Reply:
No, but only 0.2% of servers use Java 7 and they should update. Now Java 10 is here :)

I ask you on Spigot and I understand you was ok with Java 8 but if you want I can go back to Java 7

Unlike Minecraft versions, there is no reason to don't use Java 8

Reply:
you're right, of course... I kinda still live in the past sometimes, Java 8 is okay :)